libs/CCAppCommon/include/ccApplicationBase.h

```diff
@@ -44,9 +55,12 @@ class CCAPPCOMMON_LIB_API ccApplicationBase : public QApplication
 	void setupPaths();
 		
 	const QString c_VersionStr;
-	
+	// QString
```

Comment:
Well, once again, we don't need to restate the obvious, I would expect more something like 'Shader path' than 'QString' (the return type as well as the argument types clearly appear in the doxygen output).